cllc-public-app/Contexts/DynamicsExtensions.cs

```diff
@@ -143,10 +143,53 @@ public static Public.ViewModels.LegalEntity GetLegalEntityTree(this IDynamicsCli
             return result;
         }
 
+
+        public static List<MicrosoftDynamicsCRMcontact> GetLEConnectionsForAccount(this IDynamicsClient _dynamicsClient,
+            string accountId, ILogger _logger, IConfiguration _configuration, List<string> memo = null)
+        {
+            if (memo == null)
+            {
+                memo = new List<string>();
+            }
+            var result = new List<MicrosoftDynamicsCRMcontact>();
+            if (memo.Contains(accountId))
+            {
+                return result; // accountId already processed
+            }
+            memo.Add(accountId);
+
+            var filter = "_adoxio_parentaccount_value eq " + accountId;
+
+            var expand = new List<string>{
+                "adoxio_ChildProfileName_contact",
+                "adoxio_ChildProfileName_account"
+            };
+
+            var response = _dynamicsClient.Leconnections.Get(filter: filter, expand: expand);
+            var leConnections = response?.Value?.ToList();
+            leConnections.ForEach(item =>
```

Comment:
What do you think of checking for **null** here? I know most of the time we will get data back from Dynamics but in the rare case we don't this would cause a `NullPointerException` - right?

Reply:
@asanchezr  - Dynamics will either return value or an exception.  

@msehudi - please put in our usual try - catch block for a Dynamics operation to handle the exception.